VK_App/VK_App/Controller/UserImageCollectionViewController.swift

```diff
@@ -18,21 +18,35 @@ final class UserImageCollectionViewController: UICollectionViewController {
     // MARK: UICollectionViewDataSource
 
     override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        1
+        userImageName?.contentImagesName.count ?? 0
     }
 
     override func collectionView(
         _ collectionView: UICollectionView,
         cellForItemAt indexPath: IndexPath
     ) -> UICollectionViewCell {
+        let contentImage = userImageName?.contentImagesName[indexPath.row]
         guard let cell = collectionView.dequeueReusableCell(
             withReuseIdentifier: Constants.identifier,
             for: indexPath
         ) as? UserImageViewCell
         else {
             return UICollectionViewCell()
         }
-        cell.setupInfo(userImageName)
+        cell.setupInfo(contentImage)
         return cell
     }
+
+    override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let contentImageName = userImageName?.contentImagesName
+        let storyBoard = UIStoryboard(name: "Main", bundle: nil)
+        guard let userImageVC = storyBoard
+            .instantiateViewController(withIdentifier: "imageVC") as? UserImageViewController
+        else {
+            return
+        }
+        userImageVC.indexPath = indexPath.row
```

Comment:
```suggestion
        userImageVC.indexPathRow = indexPath.row
        userImageVC.indexPath = indexPath
```
indexPath хранит row и section